app/assets/javascripts/answers.js

```diff
@@ -35,8 +33,24 @@ function initUserFeedback() {
     var $button = $(ev.target).closest('.btn-feedback');
     var feedbackActive = $button.data('feedback-active');
     var feedbackString = $button.data('feedback-string');
-    var feedbackQid = $button.data('feedback-qid');
+    var feedbackQid = quizQuestions.questions[quizAllIndex].id;
     handleUserFeedback(feedbackActive, feedbackString, feedbackQid);
+
+    if (quizQuestions.questions.length > (quizAllIndex + 1)){
+      quizAllIndex = quizAllIndex + 1;
+      $('.feedback-alert').slideDown(1000, function() {
```

Comment:
Because the `$(...)` jquery calls return jquery objects, you can [chain calls](http://www.w3schools.com/jquery/jquery_chaining.asp) to make them more concise (some argue they are easier to read, but some may not; in any case, it is a convention with jquery).

Also, I find the transition pretty quick. Can we add in a slight delay, like so?
```
      $('.feedback-alert')
        .slideDown(1000, function() {
          initQuestionDisplayModalForQuizAll();
        }).delay(1000)
        .slideUp(800);
```
and
```
      $('.feedback-alert')
        .slideDown(1000)
        .delay(1000)
        .slideUp('slow', function(){
          $('#question_display_Modal').modal('hide');
        });
```